src/qt/tradehistorydialog.cpp

```diff
@@ -103,7 +94,7 @@ TradeHistoryDialog::TradeHistoryDialog(QWidget *parent) :
     ui->tradeHistoryTable->setContextMenuPolicy(Qt::CustomContextMenu);
     ui->tradeHistoryTable->setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
     ui->tradeHistoryTable->setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOn);
-    Update(); // make sure we're populated before attempting to resize to contents
+    update(); // make sure we're populated before attempting to resize to contents
```

Comment:
heh nevermind - see you changed it...  is there a "correct" way to name functions?